theory/special stars/black holes.md

```diff
@@ -7,3 +7,52 @@ nav_order: 3
 ---
 
 ## Black Holes
+
+A black hole is an astronomical object with a gravitational pull so strong that nothing, not even light, can escape it.
+
+### Structure of Black Holes
+![blackhole](../../assets/images/theory/special%20stars/black%20holes/blackhole.jpg)
```

Comment:
Please use reference from other pages on how to correctly insert credits

Reply:
Have done the changes.